### ISSUE TITLE
Detox CLI: allow overriding issued args to a test runner with the ones the user specifies after -- argument

### ISSUE THREAD

**Issue by amrew** [wix/Detox#876]
**[EDITED by @noomorph , 04.10.2018]**

For example, I want to run some Jest test based on the test case description but for now, it was impossible to override the testNamePattern parameter. Would you like to provide something like this, please:

```
detox test -c android.emu.release -- --testNamePattern myTestPattern
```

E.g., in this case the part `--testNamePattern myTestPattern` should be forwarded to the current test runner (Jest).

**Comment by burhanyilmaz** [comment]
can you write package.json? @amrew

**Comment by amrew** [comment]
i decided to create another bash file to replace `detox test` command.
then i can fill --testNamePattern param with my custom string.

This is why i can't override the parameter value.
https://github.com/wix/detox/blob/master/detox/local-cli/detox-test.js#L135

**Comment by noomorph** [comment]
I suggest adding an args passthrough feature for  `detox test ... -- --testNamePattern` similar to `npm run <script> -- --args-to-pass-through`. @rotemmiz , does it make sense for you? If yes, then pull requests are welcome.

**Comment by noomorph** [comment]
I think that this is still a relevant issue for Detox CLI, just needs rephrasing and volunteers.

**Comment by noomorph** [comment]
Fixed in `detox@^12.1.0`.
